x-pack/plugins/security_solution/server/lib/telemetry/tasks/prebuilt_rule_alerts.ts

```diff
@@ -31,11 +34,13 @@ export function createTelemetryPrebuiltRuleAlertsTaskConfig(maxTelemetryBatch: n
       const startTime = Date.now();
       const taskName = 'Security Solution - Prebuilt Rule and Elastic ML Alerts Telemetry';
       try {
-        const [clusterInfoPromise, licenseInfoPromise, packageVersion] = await Promise.allSettled([
-          receiver.fetchClusterInfo(),
-          receiver.fetchLicenseInfo(),
-          receiver.fetchDetectionRulesPackageVersion(),
-        ]);
+        const [clusterInfoPromise, licenseInfoPromise, packageVersion, alertsIndex] =
+          await Promise.allSettled([
+            receiver.fetchClusterInfo(),
+            receiver.fetchLicenseInfo(),
+            receiver.fetchDetectionRulesPackageVersion(),
+            receiver.getAlertsIndex(),
```

Comment:
not a big deal, but since `getAlertsIndex` isn't an `async` method, why do you call it this way instead of doing it as a regular method?

Reply:
Thanks! Updated in https://github.com/elastic/kibana/pull/170334/commits/c0d7fa0200c7ac7c63018283efd8f4a308170586